lib/apiservers/engine/backends/archive.go

```diff
@@ -249,9 +229,15 @@ func NewArchiveStreamWriterMap(mounts []types.MountPoint, containerDestPath stri
 		// file data.txt from local /mnt/A/data.txt will come to the persona as mnt/A/data.txt.
 		// Here, we must tell the portlayer to remove "mnt/A".  The key to determining whether to
 		// strip "A" or "mnt/A" is based on the container destination path.
-		if containerDestPath != "/" && strings.HasPrefix(aw.mountPoint.Destination, containerDestPath) {
-			aw.filterSpec.StripPath = strings.TrimPrefix(aw.mountPoint.Destination, containerDestPath)
-		}
+
+		// hickeng: the current logic is strip, then rebase for export - more efficient to collapse
+		// any reducdency here than apply to every tar entry when packing but skipping for now.
+		aw.filterSpec.StripPath = aw.mountPoint.Destination
+		aw.filterSpec.RebasePath = strings.TrimPrefix(dest, aw.mountPoint.Destination)
```

Comment:
A `RebasePath` should be tacked onto the front of an existing path, but it looks like we are taking the end of an existing path to use here.